Revert

    commit 7571fa87df453e404d8b6ca58e2da95340156849
    Author: Alexandre Julliard <julliard@winehq.org>
    Date:   Mon Aug 24 13:30:12 2020 +0200

    makefiles: Don't implicitly import the module itself when -nodefaultlibs is used.
    
    Signed-off-by: Alexandre Julliard <julliard@winehq.org>

which breaks the build with GCC 9.

Disclaimer: This is a temporary hack!

--- UTC
diff --git a/tools/makedep.c b/tools/makedep.c
index 4bd20d469d..825458156f 100644
--- tools/makedep.c
+++ tools/makedep.c
@@ -2212,11 +2212,7 @@ static struct strarray add_import_libs( const struct makefile *make, struct stra
         const char *lib = NULL;
 
         /* skip module's own importlib, its object files will be linked directly */
-        if (make->importlib && !strcmp( make->importlib, imports.str[i] ))
-        {
-            if (!is_unix) continue;
-            if (strarray_exists( &make->extradllflags, "-nodefaultlibs" )) continue;
-        }
+        if (make->importlib && !is_unix && !strcmp( make->importlib, imports.str[i] )) continue;
 
         for (j = 0; j < top_makefile->subdirs.count; j++)
         {
